lib/razorpay/payment.rb

```diff
@@ -10,6 +10,46 @@ class Payment < Entity
     def self.request
       Razorpay::Request.new('payments')
     end
+    
+    def self.createRecurringPayment(data={})
+      request.post "create/recurring" , data
+    end
+
+    def self.createJsonPayment(data={})
+      request.post "create/json" , data
+    end
+
+    def self.fetchPaymentDowntime()
```

Comment:
nit:  empty brackets not needed if no params

Reply:
Done